File: JavaScript/05. Conditional-Statements/ConditionalStatements-Homework/ConditionalStatements-Homework/05. DigitAsWord/DigitAsWord.js
/*
 * Problem 5. Digit as word

Write a script that asks for a digit (0-9), and depending on the input, shows the digit as a word (in English).
Print “not a digit” in case of invalid input.
Use a switch statement.
 */

function numberInEnglish() {
    var input = document.getElementById('number').value;
    var result = "";

    switch (input) {
        case "0":
            input += " zero";
            break;
        case "1":
            input += " one";
            break;
        case "2":
            input += " two";
            break;
        case "3":
            input += " three";
            break;
        case "4":
            input += " four";
            break;
        case "5":
            input += " five";
            break;
        case "6":
            input += " six";
            break;
        case "7":
            input += " seven";
            break;
        case "8":
            input += " eight";
            break;
        case "9":
            input += " nine";
            break;
        default:
            input += " not a digit";
            break;
    }
    console.log(input);
}

File: C# I/05. Conditional-Statements/ConditionalStatementsHomework/08. DigitAsWord/DigitAsWord.cs
using System;

class DigitAsWord
{
    /*  8. Digit as Word

        Write a program that asks for a digit (0-9), and depending on 
        the input, shows the digit as a word (in English).
        Print “not a digit” in case of invalid input.
        Use a switch statement.
    */

    static void Main()
    {
        Console.Write("Enter a digit (0-9): ");
        string digit = Console.ReadLine();
        switch(digit)
        {
            case "0":
                Console.WriteLine("Result: zero");
                break;
            case "1":
                Console.WriteLine("Result: one");
                break;
            case "2":
                Console.WriteLine("Result: two");
                break;
            case "3":
                Console.WriteLine("Result: three");
                break;
            case "4":
                Console.WriteLine("Result: four");
                break;
            case "5":
                Console.WriteLine("Result: five");
                break;
            case "6":
                Console.WriteLine("Result: six");
                break;
            case "7":
                Console.WriteLine("Result: seven");
                break;
            case "8":
                Console.WriteLine("Result: eight");
                break;
            case "9":
                Console.WriteLine("Result: nine");
                break;
            default:
                Console.WriteLine("Result: not a digit");
                break;
        }
    }
}

